### ISSUE TITLE
Apple Silicon (M1) give error at import...

### ISSUE THREAD

**Issue by dfms** [noxdafox/clipspy#62]
Hi. I have tried to make an import for an Apple Silicon M1 but it gives me error, however on the Apple with Intel I have not had problem.... could you check the configuration for macOS? Thanks and best regards.

**Comment by noxdafox** [comment]
Hello, 

which version of Python are you using? Which package did you install? What kind of error you get at import time?

**Comment by tap616** [comment]
I had a similar issue -- my system installed Python 3.9.6 didn't like the pip installation of clipspy.  I decided to try the install from source and that too had issues -- I got it down to complaining about the required library when you try to import clips:

> _clips.abi3.so, 0x0002): symbol not found in flat namespace '_ActivateRouter'

which was odd because that symbol was in the clips_source/libclips.so library but was indeed missing in all of the bult abi variants that were generated / installed.

(Note:  I had to make several tweaks to the Makefile to get that far.)

Long story short I decided to try using python from brew (as it is more up to date anyway) -- pip install worked perfectly with that and I'm up and running.

**Comment by noxdafox** [comment]
As documented in the [README](https://github.com/noxdafox/clipspy?tab=readme-ov-file#macosx) Apple Silicon is supported from Python 3.11 onwards.
